src/Adapter/Cart/CartPresenter.php

```diff
@@ -380,11 +380,20 @@ private function getTemplateVarVouchers(Cart $cart)
         $cartVouchers = $cart->getCartRules();
         $vouchers = array();
 
+        $cart_have_tax = is_null($cart->id) ? false : $cart::getTaxesAverageUsed($cart->id);
```

Comment:
Please us camel case for variables names. See also https://github.com/airbnb/javascript